src/librustc/ty/maps.rs

```diff
@@ -547,7 +553,8 @@ define_maps! { <'tcx>
     pub def_symbol_name: SymbolName(DefId) -> ty::SymbolName,
     pub symbol_name: symbol_name_dep_node(ty::Instance<'tcx>) -> ty::SymbolName,
 
-    pub describe_def: meta_data_node(DefId) -> Option<Def>
+    pub describe_def: meta_data_node(DefId) -> Option<Def>,
+    pub def_span: meta_data_node(DefId) -> Span
```

Comment:
Wait, what is `meta_data_node`? First off, it doesn't need a function, it could just be `MetaData`.
Secondly, if you have a local crate implementation, you *kind of* need a node that's not `MetaData`.
But if @nikomatsakis doesn't think this impacts incremental this can be left as-is for now.